__tests__/expr.test.ts

```diff
@@ -1178,17 +1178,43 @@ describe("expr.str", () => {
     expect(seriesActual).toFrameEqual(expected);
   });
   test("expr.replace", () => {
-    const df = pl.DataFrame({ a: [1, 2, 2, 3] });
+    const df = pl.DataFrame({ a: [1, 2, 2, 3], b: ["a", "b", "c", "d"] });
     let actual = df.withColumns(pl.col("a").replace(2, 100).alias("replaced"));
     let expected = pl.DataFrame({
       a: [1, 2, 2, 3],
+      b: ["a", "b", "c", "d"],
       replaced: [1, 100, 100, 3],
     });
     expect(actual).toFrameEqual(expected);
     actual = df.withColumns(
       pl.col("a").replace([2, 3], [100, 200], -1, pl.Float64).alias("replaced"),
     );
-    expected = pl.DataFrame({ a: [1, 2, 2, 3], replaced: [-1, 100, 100, 200] });
+    expected = pl.DataFrame({
+      a: [1, 2, 2, 3],
+      b: ["a", "b", "c", "d"],
+      replaced: [-1, 100, 100, 200],
+    });
+    expect(actual).toFrameEqual(expected);
+    actual = df.withColumns(
+      pl.col("b").replace("a", "c", "e", pl.Utf8).alias("replaced"),
+    );
+    expected = pl.DataFrame({
+      a: [1, 2, 2, 3],
+      b: ["a", "b", "c", "d"],
+      replaced: ["c", "e", "e", "e"],
+    });
+    expect(actual).toFrameEqual(expected);
+    actual = df.withColumns(
+      pl
+        .col("b")
+        .replace(["a", "b"], ["c", "d"], "e", pl.Utf8)
+        .alias("replaced"),
+    );
```

Comment:
I think it'd be a bit more readable if we took in a object/map here instead of the array. (this could totally be done in a follow up pr though. 
 
```ts
pl.col("a").replace({ 1: 11, 2: 12 }, 10, pl.Int64)
pl.col("b").replace({ a: "c", b: "d" }, "e", pl.Utf8)
```

also, i think using an object for the additional parameters would make things more readable

```ts
pl.col("a").replace({ 1: 11, 2: 12 }, {
  defaultValue: 10,
  returnType: pl.Int64,
})
```

Reply:
It would be more readable but I want to test list as param.
We can add another test using object if you like.